src/Maestro/bootstrap.ps1

```diff
@@ -26,5 +26,7 @@ if (-not $userIsMember) {
   Add-LocalGroupMember -Group $groupName -Member $userName -ErrorAction Continue
 }
 
-dotnet tool restore
-dotnet bootstrap-dnceng-configuration -r "https://vault.azure.net" -r "https://management.azure.com"
+. "$PSScriptRoot\..\..\eng\common\tools.ps1"
+InitializeDotNetCli -install:$true
```

Comment:
Just a nit, in case `install` was defined as switch in PS, the `$true` would be redundant.

Reply:
Can it be that on some weird platform or old PS this is required? I see it like this everywhere in our scripts

Reply:
To close the loop, I just checked the function's definition at `tools.ps1` and the install a regular bool parameter of a function, which means the value has to be specified. Sorry for potentially misleading comment.

`function InitializeDotNetCli([bool]$install, [bool]$createSdkLocationFile) {`